### PR TITLE
[AR] add all the sentences

### PR DESCRIPTION
**Pull request by Ahmed-farag36**

### CODE REVIEW

**Comment by synesthesiam** [comment]
@Ahmed-farag36 please add at least one test sentence for every sentence template :+1:

File: tests/ar/cover_HassCloseCover.yaml [hunk 1]
```diff
@@ -1,6 +1,35 @@
...
+
+  - sentences:
+      - "أغلق الستائر فى غرفة النوم"
+      - "أغلق ستائر غرفة النوم"
+    intent:
+      name: "HassCloseCover"
+      slots:
+        area: "bedroom"
+        device_class:
+          value:
```
Comment (balloob) [inline tests/ar/cover_HassCloseCover.yaml:32]: We don't use `value` anymore.
```suggestion
```

File: tests/ar/cover_HassOpenCover.yaml [hunk 2]
```diff
@@ -1,6 +1,36 @@
...
+
+  # Blinds
+  - sentences:
+      - "أفتح الستائر فى المطبخ"
+    intent:
+      name: "HassOpenCover"
+      slots:
+        area: "kitchen"
+        device_class:
+          value:
```
Comment (balloob) [inline tests/ar/cover_HassOpenCover.yaml:33]:
```suggestion
```

File: tests/ar/climate_HassClimateSetTemperature.yaml [hunk 3]
```diff
@@ -1,6 +1,23 @@
...
-- sentences: []
-  intent:
-    name: HassClimateSetTemperature
-    slots: {}
+  - sentences:
+      - "اضبط درجة الحرارة على 30"
+    intent:
+      name: HassClimateSetTemperature
+      slots:
+        - temperature: 30
```
Comment (balloob) [inline tests/ar/climate_HassClimateSetTemperature.yaml:8]:
```suggestion
        temperature: 30
```

**Comment by alamoudimoh** [comment]
Hi Ahmed,

How can we help?

**Comment by Ahmed-farag36** [comment]
Would make a new pull request

**Pull request closed without merging**